src/index.js

```diff
@@ -0,0 +1,106 @@
+let index = 0;
+function toggleLike(itemId) {
+  const heartIcon = document.getElementById(`${itemId}-heart`);
+  const likeCount = document.getElementById(`${itemId}-likes`);
+
+  // Check if the item is liked
+  const isLiked = heartIcon.classList.contains('fas');
+
+  if (isLiked) {
+    heartIcon.classList.remove('fas');
+    heartIcon.classList.add('far');
+    likeCount.textContent = parseInt(likeCount.textContent, 10) - 1;
+  } else {
+    heartIcon.classList.remove('far');
+    heartIcon.classList.add('fas');
+    likeCount.textContent = parseInt(likeCount.textContent, 10) + 1;
+  }
+}
+
+// List of movie IDs
+const movieIds = ['169', '82', '431'];
```

Comment:
[OPTIONAL] It is better to have more movies to display but you can keep like this for now:)